vignettes/socialmixr.Rmd

```diff
@@ -478,7 +490,7 @@ ggplot(df, aes(x = age.group, y = age.group.contact, fill = contacts)) +
 
 # Plotting (R base)
 
-The contact matrices can be plotted with the `matrix_plot()` function as a grid of colored rectangles with the numeric values in the cells. Heat colors are used by idefault, though this can be changed.
+The contact matrices can be plotted with the `matrix_plot()` function as a grid of colored rectangles with the numeric values in the cells. Heat colors are used by default, though this can be changend.
```

Comment:
fixed one typo, introduced another?

Reply:
yes... sorry! fixed this in my latest commit